Examples/SpatialObjects/SpatialObjectTransforms.cxx

```diff
@@ -22,53 +22,65 @@
-// invertible when assigned, or the assignment fails.
-//
-// WorldSpace
+// \item[ObjectToParentTransform]. SpatialObjects have only one transform
+// that they directly control, their
+// \code{ObjectToParentTransform}. This transform specifies how an object's
+// \code{ObjectSpace} is
+// transformed to fit into its parent's \code{ObjectSpace}. The
+// \code{ObjectToParentTransform} is an affine transform, and it is confirmed
+// to be invertible when assigned, or the assignment fails.
 //
-// WorldSpace is not directly controlled by any SpatialObject except the
+// \item[WorldSpace]. \code{WorldSpace} is not directly controlled by any
+// SpatialObject except the
 // SpatialObject at the top level of the parent-child tree hierarchy of
 // Spatial Objects. That is, any SpatialObject that does not have a parent
-// exists in a WorldSpace that is defined by applying its
-// ObjectToParentTransform to its ObjectSpace.
+// exists in a \code{WorldSpace} that is defined by applying its
+// \code{ObjectToParentTransform} to its \code{ObjectSpace}.
+//
+// \end{description}
 //
 // Several member functions and variables are available to every SpatialObject
 // so that they can readily access the WorldSpace in which they exist:
 //
-// * ProtectedComputeObjectToWorldTransform: Composes its ObjectToParentTransform
-// with its parent's cached ObjectToObjectToWorldTransform, to determine the
-// transform from the object's ObjectSpace to WorldSpace.   This transform is
+// \begin{description}
+//
+// \item[ProtectedComputeObjectToWorldTransform()]: This function is called
+// whenever \code{Update()} is called.  It composes the object's
+// \code{ObjectToParentTransform}
+// with its parent's cached \code{ObjectToWorldTransform}, to determine the
+// transform from the object's \code{ObjectSpace} to \code{WorldSpace}.
+// This transform is
 // always invertible.   This call will cause all children objects to also
-// update their cached ObjectToWorldTransform.   This function should be
-// called on the top level object once all children object's
-// ObjectToParentTransforms have been set.   This function should be called
-// on children objects when their ObjectToParentTransforms have been changed.
-// Due to the potentially high computational cost of this function, this
-// function must be manually called by users.
-//
-// * GetObjectToWorldTransform: Returns the cached ObjectToWorldTransform.
-// It is the user's responsibility to call ProtectedComputeObjectToWorldTransform when
-// necessary, prior to calling GetObjectToWorldTransform, otherwise the
+// update their cached \code{ObjectToWorldTransform}.   This function should be
+// called on the top level object (via \code{Update()}) once all children
+// object's
+// \code{ObjectToParentTransforms} have been set.   This function should
+// be called
+// on children objects when their \code{ObjectToParentTransforms} have been
```

Comment:
\code{ObjectToParentTransforms} -> \code{ObjectToParentTransform}s
Plural s outside of `\code`

Reply:
Fixed here and elsewhere.